### PR TITLE
fix: better logging.

### PR DESCRIPTION
**Pull request by neotyk**

### CODE REVIEW

**Review by sid597 — APPROVED** [review]

File: test/athens/common_events/utils_test.cljc [hunk 1]
```diff
@@ -0,0 +1,30 @@
+(ns athens.common-events.utils-test
+  (:require
+    [athens.common-events                 :as sut]
```
Comment (filipesilva) [inline test/athens/common_events/utils_test.cljc:3]: `sut`?

File: src/clj/athens/self_hosted/clients.clj [hunk 2]
```diff
@@ -70,24 +71,26 @@
...
-                   ", status:" (:event/status data))
+                   (if type
+                     (str ", type: " type)
+                     (str ", status: " status)))
         (http/send! channel (->transit data)))
       ;; TODO internal failure mode, collect in reporting
       (log/error "Not sending invalid event to username:" username
                  ", event-id:" (:event/id data)
-                 ", type:" (:event/type data)
+                 ", type:" (common-events/find-event-or-atomic-op-type data)
```
Comment (filipesilva) [inline src/clj/athens/self_hosted/clients.clj:85]: `find-event-or-atomic-op-type` doesn't seem to have defined behaviour or tests for invalid events, and events that fall here are invalid in one way or another.

**Review by filipesilva — APPROVED** [review]

File: test/athens/common_events/utils_test.cljc (hunk 1 above)
Comment (neotyk) [reply test/athens/common_events/utils_test.cljc:3]: Suit Under Test

**Pull request merged**